apps/interpreter/src/runtime-parameter-util.spec.ts

```diff
@@ -3,36 +3,35 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 
 import {
-  PropertyValuetype,
+  PrimitiveValuetypes,
   RuntimeParameterLiteral,
-  runtimeParameterAllowedForType,
 } from '@jvalue/jayvee-language-server';
 
 import { parseParameterAsMatchingType } from './runtime-parameter-util';
 
 describe('runtime parameter utils', () => {
   describe('parameter parsing', () => {
-    Object.keys(PropertyValuetype).forEach((propertyValuetypeKey) => {
-      const propertyValuetype =
-        PropertyValuetype[
-          propertyValuetypeKey as keyof typeof PropertyValuetype
+    Object.keys(PrimitiveValuetypes).forEach((primitiveValueTypeKey) => {
+      const primitiveValuetype =
+        PrimitiveValuetypes[
+          primitiveValueTypeKey as keyof typeof PrimitiveValuetypes
         ];
```

Comment:
I think this can be simplified now:
```suggestion
    Object.values(PrimitiveValuetypes).forEach((primitiveValuetype) => {
```